src/App.tsx

```diff
@@ -21,6 +44,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={(event) => onSelect(+event.target.value)}
```

Comment:
let's handle `even.target.value` in our `onSelect` function 
```suggestion
            onChange={onSelect}
```